.github/workflows/dev_version_script.py

```diff
@@ -17,61 +17,68 @@
 
 import re
 
-VERSION_FILE_PATH = 'pennylane_lightning/_version.py'
+VERSION_FILE_PATH = "pennylane_lightning/_version.py"
```

Comment:
Is there an advantage to using `" "` rather than `' '` throughout? Or did this come as a result of linting?

Reply:
No advantage. I figured a few Python scripts had not had the `black` update, including `setup.py`, so I thought I would pass Black on them to help `git diff` figure out what really changes in the future.

Reply:
Sounds good, thanks :+1: